amt.go

```diff
@@ -295,6 +306,50 @@ func (n *Node) forEachAt(ctx context.Context, bs cbor.IpldStore, height int, sta
 
 }
 
+func (n *Node) parForEachAt(ctx context.Context, bs cbor.IpldStore, height int, start, offset uint64, parNum int, cb func(uint64, *cbg.Deferred) error) error {
+	errCh := make(chan struct{})
+	toProcess := make(chan struct{}, parNum)
+	for i := 0; i < parNum; i++ {
+		toProcess <- struct{}{}
+	}
```

Comment:
I don't understand what this channel is for.  It appears to be a some kind of source token bucket, but... the `wg sync.WaitGroup` further down would appear to also be doing the exact same thing (and is a more appropriate tool for that purpose).